src/iter.zig

```diff
@@ -3,31 +3,63 @@ const cs = @import("capstone-c");
 const Insn = @import("insn.zig").Insn;
 const Handle = cs.csh;
 
-// The Iterator for traversing the disassembler
+/// The Iterator for traversing the disassembler
 pub const Iter = struct {
```

Comment:
To be consistent with the naming throughout also zig's std:
```suggestion
pub const IterUnamanged = struct {
```